akka-http/src/main/scala/akka/http/javadsl/marshalling/Marshaller.scala

```diff
@@ -88,8 +88,8 @@ object Marshaller {
    * Helper for creating a "super-marshaller" from a number of "sub-marshallers".
    * Content-negotiation determines, which "sub-marshaller" eventually gets to do the job.
    */
-  def oneOf[A, B](ms: Marshaller[A, B]*): Marshaller[A, B] = {
-    fromScala(marshalling.Marshaller.oneOf[A, B](ms.map(_.asScala): _*))
+  def oneOf[A, B](m1: Marshaller[A, B], m2: Marshaller[A, B]): Marshaller[A, B] = {
```

Comment:
In Java you'd then get the "heap pollution" warning at call sites, afair. I don't think I could reliably remove that in Java code, let alone scala.

My current version:
```scala
  @SafeVarargs @varargs def oneOf[A, B](ms: Marshaller[A, B]*): Marshaller[A, B] = {
    fromScala(marshalling.Marshaller.oneOf[A, B](ms.map(_.asScala): _*))
  }
```
generates compiler warnings at all call sites:
```
[warn] ..../akka-http-tests/src/test/java/akka/http/javadsl/server/JavaRouteTest.java:78: unchecked generic array creation for varargs parameter of type akka.http.javadsl.marshalling.Marshaller<java.util.UUID,akka.http.javadsl.model.RequestEntity>[]
[warn]     Marshaller.oneOf(
```
A (small) advantage of the n-arity methods would be that we can introduce type variance...